src/Shared/PlatformSupport/StandardRuntimePlatform.cs

```diff
@@ -12,12 +11,16 @@ namespace Avalonia.Shared.PlatformSupport
 {
     internal partial class StandardRuntimePlatform : IRuntimePlatform
     {
+
+#if NETSTANDARD
+        public void PostThreadPoolItem(Action cb) => ThreadPool.QueueUserWorkItem(_ => cb(), null);
+#else
```

Comment:
There are already different implementations of the other parts of this class in the `Avalonia.DotNetFrameworkRuntime` and `Avalonia.DotNetCoreRuntime` libraries. Rather than using an `#if` here shouldn't the relevant parts be moved there?